### PR TITLE
create volunteers table

### PR DESCRIPTION
**Pull request by johndeyrup**
First of two part PR to create volunteers and their associated vaccine sites.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 require('dotenv').config();
-require('newrelic');
-require('@newrelic/aws-sdk');
+
 const geocoder = require('google-geocoder');
 var cluster = require('cluster');
 
+if (process.env.NODE_ENV === 'production') {
```
Comment (johndeyrup) [inline app.js:6]: cc @mzagaja

File: db/volunteers.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        const { rows } = await pool.query('INSERT INTO volunteers (email, firstName, lastName) VALUES ($1, $2, $3)', [volunteer.email, volunteer.firstName, volunteer.lastName]);
+        return rows;
+    } catch (error) {
+        console.error(`an error occurred when creating a new volunteer ${volunteer}`, volunteer);
+        throw new Error(`An unexpected error occurred creating a new volunteer error: ${error}`);
+    }
+};
+
+/**
+* Upate volunteer email
```
Comment (jonhines) [inline db/volunteers.js]: nit: typo

**Review by jonhines — APPROVED** [review]

**Review by mwellman17 — COMMENTED** [review]
Can we use the volunteer ID as a unique API key or can we add a key to the volunteer object that can be queried?

File: dbschema.sql [hunk 3]
```diff
@@ -64,4 +64,15 @@ EXECUTE FUNCTION trigger_set_timestamp();
...
+CREATE TABLE volunteers (
+  id               INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  email             VARCHAR
+                   (1000) UNIQUE NOT NULL,
+  firstName       VARCHAR
+                   (1000)       NOT NULL,
+  lastName        VARCHAR
+                   (1000)       NOT NULL,
+  lastupdated      TIMESTAMP    NOT NULL DEFAULT NOW()
+);
```
Comment (mwellman17) [inline dbschema.sql]: Let's add an admin boolean so the front end can give access to volunteer management.

**Pull request merged**